e2e/app/page-mixin/PageNavigation.ts

```diff
@@ -0,0 +1,95 @@
+import {Page} from 'puppeteer';
+import {iconXpath} from '../aou-elements/xpath-defaults';
+import {findIcon} from '../aou-elements/xpath-finder';
+
+
+export const LINK = {
+  HOME: 'Home',
+  ADMIN: 'Admin',
+  USER_ADMIN: 'User Admin',
+  PROFILE: 'Profile',
+  SIGN_OUT: 'Sign Out',
+  CONTACT_US: 'Contact Us',
+  USER_SUPPORT: 'User Support',
+  YOUR_WORKSPACES: 'Your Workspaces',
+  FEATURED_WORKSPACES: 'Featured Workspaces',
+};
+
+export const LINK_ICON = {
+  HOME: 'home',
+  ADMIN: 'user',
+  CONTACT_US: 'envelope',
+  USER_SUPPORT: 'help',
+  YOUR_WORKSPACES: 'applications',
+  FEATURED_WORKSPACES: 'star',
+};
+
+
+export default class PageNavigation {
```

Comment:
Are these really mixins? Aren't they just base classes? Mixin implies I could include multiple of these into a page - I don't see how you would be able to do that in the current inheritance model.